examples/single-node/README.md

```diff
@@ -0,0 +1,9 @@
+This example portrays a single-node configuration, where all nodes are deployed on the same machine. In this deployment:
+ - `vitellone` is the hostname where `trento-server` gets deployed
+
+Agent nodes are *not* deployed with this configuration.
+
+Two variants are available:
+ - vars-with-alerting.json: In this example, we set up alerting and set the SMTP configuration. With this, important alerts should arrive at the
```

Comment:
Just a suggestion: 

~~With this, important~~, alerts should arrive at the --> Alerts should arrive at ....

It sounds good, but in the end these are empty words. Also as a reader I start to ask my self, what about the unimportant ? :D